.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint GitHub Actions workflows
+
+on:
+  push:
+    paths:
+      - ".github/workflows/*.ya?ml"
+  pull_request:
+    paths:
+      - ".github/workflows/*.ya?ml"
+
+jobs:
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Download actionlint
+        id: get_actionlint
+        run: bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
```

Comment:
Let's pin a specific hash instead of using `main`, _just in case_.

We're just `curl | bash`-ing here, and I'd feel safer if it's a specific, reproducible, probably-not-pwned version of it.